From 94dc667b9b39b37c56af0a81644f19cb83f974d3 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 05 Jun 2022 16:08:39 +0300
Subject: [PATCH] Revert "[SW-77251] fit: gaudi3: add copy image function"

Revert rom recovery implementation for secured firmware update via
inband and OOB.

This reverts commit 342ce6923188c19f7689fd024929e0380c5db2de.

Change-Id: If220c7fb2c4ec79d8213ef5f285b6e7daeefe586
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index e85a85d..9d1ed5f 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -880,13 +880,6 @@
 	image_entry();
 }
 
-void arcboot_copy_image(uint64_t dst, const void *buf, int len, __unused const char *arch)
-{
-	uint32_t arc_dst = (uint32_t)arcboot_rom_get_arc_addr(dst);
-
-	memcpy((void *)arc_dst, buf, len);
-}
-
 void arcboot_rom_in_recovery(void)
 {
 	uint8_t override = 1;
